docs/documentation-standards/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: doc
+title: Documentation Standards
+description: Documentation Standards for WP-CLI commands.
+category: References
+---
+
+We have set some Documentation Standards for WP-CLI commands.
+
+## Examples
+
+### Subcommand
+
+* Blank line should be there before and after for each example.
+* Each example should have 3 parts.
+    - Description
+        + Must start with `#` and a space.
+        + Sentence should start with capital letter
+        + Comment must end in full-stops, exclamation marks, or question marks.
+        + Eg: `# Create database.`
+    - Command
+        + Must start with `$` and a space. Eg: `$ wp db create`
+    - Sample Output
+        + Keep exact output. Note spaces and indentation in the output. Exception: If output is very long, it could be truncated to show suitable section only.
+        + Eg: `Success: Database created.`
+* If possible, keep at least two examples for each command. One showing basic use and another advanced use. More use cases the better.
```

Comment:
For each of these standards, it would be helpful to include _why_. For instance, why is it important for there to be a blank line before and after each example?